src/service/iot_config.proto

```diff
@@ -22,45 +22,43 @@ import "region.proto";
 //
 // == DevAddrs ==
 //
-// - `devaddr_ranges` are inclusive on both sides, `start_addr` and `end_addr`.
+// - `devaddr_range` are inclusive on both sides, `start_addr` and `end_addr`.
 //
-// - `org_res_v1.devaddr_ranges` provides the ranges of DevAddrs available to
+// - `org_res_v1.devaddr_range` provides the ranges of DevAddrs available to
 //    any `route_v1` under an Org.
 //
-// - `route_v1.devaddr_ranges` provides the ranges of DevAddrs that should go to
+// - `route_v1.devaddr_range` provides the ranges of DevAddrs that should go to
 //    this specific Route. This ranges must always fall within the
-//    `devaddr_ranges` of the owning org.
+//    `devaddr_range` of the owning org.
 
 enum action_v1 {
-  create = 0;
-  update = 1;
-  delete = 2;
+  add = 0;
+  remove = 1;
 }
 
 // Define an organisation
 message org_v1 {
   uint64 oui = 1;
-
   // Org admin key
   bytes owner = 2;
   // Key only used for DC payments
   bytes payer = 3;
   // List of keys allowed some specific actions, see services.
   repeated bytes delegate_keys = 4;
-  // Use req timestamps to validate order/avoid replay
-  uint64 nonce = 5 [ deprecated = true ];
 }
 
 // Device address range, ex: 16#00000001 to 16#0000000A
 message devaddr_range_v1 {
-  uint32 start_addr = 1;
-  uint32 end_addr = 2;
+  string route_id = 1;
```

Comment:
`devaddr_range_v1` is also used to communicate devaddr restrictions for orgs, which is outside the scope of a route.

Do we want to have 2?
`devaddr_range_v1` that applies to routes, and `devaddr_constraint_v1` that is just a start and end? and could evolve to have an `org_id` if we want to go that route?

Reply:
I remember you asking about a separate `constraint` type and now I think it makes sense here that I understand the domain more and we’ve gone down this route in the refactor